book/src/keymap.md

```diff
@@ -373,3 +375,34 @@ Keys to use within prompt, Remapping currently not supported.
 | `BackTab`               | Select previous completion item                                         |
 | `Enter`                 | Open selected                                                           |
 
+# File explorer
```

Comment:
Should this be called tree explorer rather than file explorer? Given that later tree explorer functionality may be reused for something else.

Reply:
Just my two cents, but I have my doubts...

"tree explorer" sounds like a name which could be used _in the code_ because - as you correctly point out - it could share its code base for different purposes. However, IMHO that name should _not_ be presented to the user:

The user explores... files. Not trees. :deciduous_tree:

Add to that the fact that "file explorer" is a pretty common term, e.g. in desktop environments, but also in other editors/IDEs.

If we really want to use the word "tree", I could imagine the name "file tree", but I still find "file explorer" more intuitive. :)